src/components/people-grid/people-grid.jsx

```diff
@@ -0,0 +1,46 @@
+const PropTypes = require('prop-types');
```

Comment:
All of the people-grid files are in a separate PR that Ben is reviewing, so I don't think we need to look to closely at them here